### ISSUE TITLE
Crashes on Ubuntu

### ISSUE THREAD

**Issue by zigzag-way** [yegorus/node-bestchange#2]
It crashes on Ubuntu (Digital Ocean), looks like it happens when unpacking archive, but Im not sure. I dont see a way to debug it correctly to understand whats wrong.
On Windows 11 everything works fine.

**Comment by yegorus** [comment]
Hi, @zigzag-way!

I refactored the package in 1.0.4 version, if possible, check it works now?
https://www.npmjs.com/package/node-bestchange/v/1.0.4

Test on ubuntu passed https://github.com/yegorus/node-bestchange/actions/runs/3998977681/jobs/6862296466 (in tests mock zip archive from fs)

Please be careful, the initialization construct has changed:
```js
async (() => {
    const Api = require('node-bestchange')
    const api = await (new Api()).load(); // -- new Api(<url> default = 'http://api.bestchange.ru/info.zip')
})()
```

PR with fix: https://github.com/yegorus/node-bestchange/pull/4

Thank you for the feedback!

**Comment by zigzag-way** [comment]
Hi,

Sorry for bothering you. Finally it turned out to be a "not enough memory" problem on my Digital Ocean droplet. 512mb is not enough for this line of code and the process gets killed:

`iconv.encode(iconv.decode(data, 'windows-1251'), 'utf8').toString())`

So everything is working fine on Ubuntu if you have enough memory.

**Comment by yegorus** [comment]
I have published version v1.0.5 with some fix memory leak, but now the usage is about 90 mb and it is not very good.
https://www.npmjs.com/package/node-bestchange/v/1.0.5

The main place where memory is consumed:
https://github.com/yegorus/node-bestchange/blob/master/src/collections/Rates.js#L12 (parsing file 22mb)
so far there are no ideas how to improve it.

**Comment by zigzag-way** [comment]
Have you seen my Pull Req about Rates?

**Comment by yegorus** [comment]
@zigzag-way No, I don't see it. Please give me a link PR?

**Comment by yegorus** [comment]
Hi, i have published your PR 1.0.6
https://www.npmjs.com/package/node-bestchange/v/1.0.6
